content/cybersecurity/concepts/nmap/terms/os-detection/os-detection.md

```diff
@@ -0,0 +1,76 @@
+---
+Title: 'OS Detection'
+Description: 'One of the best-known features of Nmap is remote OS detection using TCP/IP stack fingerprinting.'
```

Comment:
```suggestion
Description: 'Remotely scans a target host and presents details of its operating system if there is a match.'
```

For term entries in Docs, let's set up the `Description` such that it begins with a present-tense verb 😄